pkg/acsengine/k8s_versions.go

```diff
@@ -24,6 +24,7 @@ var k8sComponentVersions = map[string]map[string]string{
 		"aci-connector":                    "virtual-kubelet:latest",
 		ContainerMonitoringAddonName:       "oms:ciprod05082018",
 		AzureCNINetworkMonitoringAddonName: "networkmonitor:v0.0.4",
+		"cluster-autoscaler":               "cluster-autoscaler:v1.3",
```

Comment:
Is v1.3 the image tag? If so, I think v1.3.0 should be used here.

The original image released is `gcr.io/google_containers/cluster-autoscaler:v1.3.0`.

Reply:
thanks! updated tag